AutoscoperM/AutoscoperM.py

```diff
@@ -454,15 +453,13 @@ def onGeneratePartialVolumes(self):
                 mainOutputDir=mainOutputDir,
                 volumeSubDir=tiffSubDir,
                 transformSubDir=tfmSubDir,
-                trackingSubDir=trackingSubDir,
                 modelSubDir=modelSubDir,
             )
 
             self.logic.createPathsIfNotExists(
                 mainOutputDir,
                 os.path.join(mainOutputDir, tiffSubDir),
                 os.path.join(mainOutputDir, tfmSubDir),
-                os.path.join(mainOutputDir, trackingSubDir),
```

Comment:
suggest leaving the Tracking directory. Autoscoper default .tra output dir

Reply:
Sounds good, done.